README.md

```diff
@@ -209,7 +209,7 @@ Use .editorconfig | <span style="color:green">small</span> | ✅ | ✅ | ✅ | 
 Format your code automatically | <span style="color:green">small</span> | ❌ | ✅ | ❌ | ❌ | ❌
 Use ESLint | <span style="color:yellow">medium</span> | ❌ | ✅ | ❌ | ❌ | ❌
 ESLint Without Errors | <span style="color:yellow">medium</span> | ❌ | ✅ | ❌ | ❌ | ❌
-Use a different linter | <span style="color:yellow">medium</span> | ❌ | ✅ | ❌ | ❌ | ❌
+Use a different linter | <span style="color:yellow">medium</span> | ❌ | ✅ | ❌ | ✅ | ❌
```

Comment:
This is specifically for Deprecated TS Lint. Also, linter practice for python checks whether the user uses any linter, not that he uses a bad linter and he should use a different one. I think the best solution here would be to rename (just in readme) the practice `Use ESLint` (line 210) to `Use a Linter` and have the checkmarks for each language where this practice is implemented there. Do you agree? :)

Reply:
Ohh I didn't know that yeah I guess considering linter practice for all languages as one instead of ESLint for ts and js is better will make the needed changes